fusor-ember-cli/bower.json

```diff
@@ -4,7 +4,7 @@
     "ember": "1.11.1",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
     "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-data": "1.0.0-beta.16.1",
+    "ember-data": "~1.13.0",
```

Comment:
I think beta.16 would still work, but let's go with the fix stable release version for ember-data that they called `1.13`